core/src/com/unciv/ui/pickerscreens/PickerScreen.kt

```diff
@@ -88,4 +89,16 @@ open class PickerScreen(disableScroll: Boolean = false) : BaseScreen() {
     fun removeRightSideClickListeners() {
         rightSideButton.clearListeners()
     }
+
+    companion object {
+        /** Icon size used in [getPickerOptionButton]. */
+        const val pickerOptionIconSize = 30f
+        /** Return a button for picker screens that display a list of big buttons with icons and labels. */
+        fun <I: Actor?> getPickerOptionButton(icon: I, label: String): Button {
```

Comment:
Here too we don't need renovating, we can accept an Actor

Reply:
Right. Pitfalls of automatic refactor. I skimmed the diff a couple times to try to find needed changes, but clearly missed this. :)